### PR TITLE
General improvements to manual pages

### PR DESCRIPTION
**Pull request by SuperSonicHub1**
Attempting a good first pull request again by starting *real* small.

- Moved multiple pages to more appropriate sections
- Added "Open" links
- Added and edited gists (command - \<gist\>)
- Edited descriptions

### CODE REVIEW

File: Base/usr/share/man/man8/TelnetServer.md [hunk 1]
```diff
@@ -4,22 +4,23 @@ TelnetServer - Serenity telnet server
 
 ## Synopsis
 
-```**sh
-# TelnetServer [options]
+```sh
+$ TelnetServer [-p port] [-c command]
```
Comment (alimpfard) [inline Base/usr/share/man/man8/TelnetServer.md:8]: I doubt this runs as a non-root user.

File: Base/usr/share/man/man8/WebServer.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Name
 
-WebServer
+WebServer - Serenity web server
```
Comment (alimpfard) [inline Base/usr/share/man/man8/WebServer.md:3]: ...eh.
doesn't sound that useful or good, it's literally the program name

**Review by alimpfard — COMMENTED** [review]
- Please use the imperative mood for commit messages (see https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy).
- Not necessarily a requirement, but this feels a bit _too_ split up, perhaps group related commits a bit?
    e.g. Base: Add gists to manpages that do not have one

File: Base/usr/share/man/man8/WebServer.md (hunk 2 above)
Comment (SuperSonicHub1) [reply Base/usr/share/man/man8/WebServer.md:3]: > doesn't sound that useful or good, it's literally the program name
All the man pages are like that. Did it for consistency.

File: Base/usr/share/man/man8/TelnetServer.md (hunk 1 above)
Comment (SuperSonicHub1) [reply Base/usr/share/man/man8/TelnetServer.md:8]: You are correct. That should be added to the man page.

**Comment by SuperSonicHub1** [comment]
@alimpfard Sorry for taking so long to incorporate fixes; life got in the way.

**Comment by SuperSonicHub1** [comment]
Thanks for accepting my pull!

**Pull request merged**